functions.R

```diff
@@ -0,0 +1,84 @@
+## Functions for target workflow to look at lake Mendota temperature model errors
+
+## Function for pulling down data from 
+## USGS's ScienceBase https://www.sciencebase.gov/
+download_data <- function(dataID, fetch_dir, file_name){
+  if(dir.exists(fetch_dir) == F){
+    dir.create(fetch_dir)
+  }
+  
+  path <- file.path(fetch_dir, file_name)
+  item_file_download('5d925066e4b0c4f70d0d0599', 
```

Comment:
```suggestion
  item_file_download(dataID, 
```
This will use the `dataID` specified in your targets call - which is the same ID but now your argument will do something in the function and isn't hard coded